library/std/src/thread/local.rs

```diff
@@ -452,7 +478,7 @@ impl<T: 'static> LocalKey<RefCell<T>> {
     /// Panics if the key currently has its destructor running,
     /// and it **may** panic if the destructor has previously been run for this thread.
     ///
-    /// # Example
+    /// # Examples
```

Comment:
But it's still only one example?

Reply:
It's how it's done in all std/core docs. I can't find where the rule is described though... Take a look at the [String docs](https://doc.rust-lang.org/nightly/std/string/struct.String.html) for example.

So even if there is only one code example, it's supposed to be titled "Examples".